core/src/classes/codeConfig.ts

```diff
@@ -147,6 +147,32 @@ export function extractNonNullParts(
   return cleanedOptions;
 }
 
+export function getAutoBootstrappedProperty(
+  sourceConfigObject: ConfigurationObject,
+  otherConfigObjects: ConfigurationObject[]
+) {
+  if (sourceConfigObject.class?.toLowerCase() !== "source") return null;
+  if (!sourceConfigObject.mapping) return null;
+
+  const mappingValues = Object.values(sourceConfigObject["mapping"]);
+  for (const value of mappingValues) {
```

Comment:
should we do the mapping story as part of this? at least the source one? https://www.pivotaltracker.com/story/show/177734437

Reply:
hard to say but wanted to bring up we have something in this same area.

Reply:
We'll do that story soon after